website/docs/instantiate_config.md

```diff
@@ -0,0 +1,96 @@
+---

```

Comment:
### **This is a general comment for all the next documentation. **

```
“You cannot teach a man anything; you can only help him find it within himself.”
― Galileo Galilei
```

Although it's a technical article. Humans are emotional creatures who understand things through emotion. through stories.

Creating a class with a meaningful name and meaning will be easier to understand and connect with. 
for example if ill replace the `ExampleClass` with `UserCoreLib`

```
class UserCoreLib(CoreLib):
   def __init__(config: DoctConfig):
    self.config = config
    self.authenticate_client = instantiate_config(self.config.core_lib.client.auth)
    # Here you will add more 2 meaningful classes that the UserCoreLib needs
   self.<pleaseFill1> =  = instantiate_config(self.config.core_lib....
   self.<pleaseFill1> =  = instantiate_config(self.config.core_lib....   
```

Reply:
Thankyou, will follow this from now on